src/app/service/tool-group/tool-group.service.ts

```diff
@@ -0,0 +1,285 @@
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import { JsonApiDataStore } from 'jsonapi-datastore';
+import { countries, ICountry } from 'countries-list';
+import { AuthService } from '../auth/auth.service';
+import {
+  LanguageBCP47Service,
+  LanguageBCP47,
+} from '../../service/languages-bcp47-tag.service';
+import { AbstractService } from '../abstract.service';
+import { ToolGroup, RuleTypeEnum } from '../../models/tool-group';
+import { environment } from '../../../environments/environment';
+
+interface PraxisData {
+  confidence: string[];
+  openness: string[];
+}
+@Injectable()
+export class ToolGroupService extends AbstractService {
+  constructor(
+    private http: Http,
+    private authService: AuthService,
+    private languageBCP47Service: LanguageBCP47Service,
+  ) {
+    super();
+  }
+  private readonly toolGroupsUrl = environment.base_url + 'tool-groups';
+
+  praxisConfidentData = {
+    0: {
+      name: 'Very confident',
+    },
+    1: {
+      name: 'Somewhat confident',
+    },
+    2: {
+      name: 'Neutral',
+    },
+    3: {
+      name: 'Not very confident',
+    },
+    4: {
+      name: 'Not confident at all',
+    },
+  };
+
+  praxisOpennessData = {
+    0: {
+      name: 'Very Open',
+    },
+    1: {
+      name: 'Somewhat open',
+    },
+    2: {
+      name: 'Neutral',
+    },
+    3: {
+      name: 'Not very open or interested',
+    },
+    4: {
+      name: 'Not open at all',
+    },
+  };
+
+  getToolGroups(): Promise<ToolGroup[]> {
+    return this.http
+      .get(this.toolGroupsUrl, this.authService.getAuthorizationAndOptions())
+      .toPromise()
+      .then((response) => new JsonApiDataStore().sync(response.json()))
+      .catch(this.handleError);
+  }
+
+  getToolGroup(
+    id: number,
+    include = 'rules-language,rules-praxis,rules-country,tools,tools.tool',
```

Comment:
```suggestion
    include = 'rules-language,rules-praxis,rules-country,tools.tool',
```

the `tools` include is redundant since it is already being included by `tools.tool`